pootle/templates/help/quality_checks.html

```diff
@@ -4,18 +4,33 @@
 {% block header_block %}{% endblock %}
 
 {% block content %}
+
+<head>
```

Comment:
We can't put `<head>` here. It is already defined in the layout.html template. Please see the `css_overrides` block which you can override in this template:
https://github.com/evernote/zing/blob/master/pootle/templates/layout.html#L18